alembic.ini

```diff
@@ -0,0 +1,73 @@
+# Uber specific alembic configuration
+
+[alembic]
+# The script_location is derived from the uber configuration, so this value is
+# completely ignored.
+# IGNORED
+script_location = alembic
```

Comment:
Oh yeah, I could easily remove `alembic.ini` altogether. It was definitely an intentional choice to leave it in, because I figured developers would be more familiar with that usage. The only caveat is that if you *DO* decide to use `alembic.ini`, alembic will complain if you remove `script_location`.